[2883][ADD] website_sale_address_email_readonly (#225)
qrtl/mi7-custom: website_sale_address_email_readonly/readme/DESCRIPTION.rst

This module does the following:

> - Make readonly email field in address form of website sale if the user has email address.

Reviewer: ```suggestion
- Makes the address form email field in eCommerce checkout read-only if the user has email address.
```
